### PR TITLE
rollbar v1 port

### PR DESCRIPTION
**Pull request by iwex**

### CODE REVIEW

**Comment by iwex** [comment]
I'll rewrite tests soon

File: src/RollbarLogHandler.php [hunk 1]
```diff
@@ -46,98 +40,102 @@ class RollbarLogHandler extends AbstractLogger
...
-                $this->rollbar->person = $context['person'];
+                $person = $context['person'];
                 unset($context['person']);
-            } else {
-                if ($this->rollbar->person_fn && is_callable($this->rollbar->person_fn)) {
-                    $data = @call_user_func($this->rollbar->person_fn);
-                    if (isset($data['id'])) {
-                        $this->rollbar->person = call_user_func($this->rollbar->person_fn);
-                    }
-                }
```
Comment (drbyte) [inline src/RollbarLogHandler.php]: Why remove support for `person_fn`?  One of the recent issues in this repo led to people specifically using the `person_fn` functionality.

Comment (iwex) [inline src/RollbarLogHandler.php]: https://github.com/rollbar/rollbar-php/blob/master/src/DataBuilder.php#L618 is it not the same?

Comment (drbyte) [inline src/RollbarLogHandler.php]: Okay - I missed seeing that 👍

**Pull request closed without merging**